registrations/views.py

```diff
@@ -213,3 +213,23 @@ def post(self, request, *args, **kwargs):
             status = 400
 
         return Response(resp, status=status)
+
+
+class PersonnelCodeView(APIView):
+
+    """ PersonnelCodeView Interaction
+        GET - returns a list of identities with personnel codes
```

Comment:
This doesn't seem to be correct. It returns a list of personnel codes, not identities.